client/actions/documentActions.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Documents action, disptach action and
+ * action types of each action to the reducer
+ */
+
+import axios from 'axios';
+import types from './actionTypes';
+
+/**
+ *
+ * loaddocumentsuccess
+ * @export
+ * @param {any} documents  returned documents from api call
+ * @returns {any} action and action types
+ */
+export function loadDocumentSuccess(documents) {
+  return { type: types.LOAD_DOCUMENT_SUCCESS, documents };
+}
+
+/**
+ * create new document success action
+ *
+ * @export
+ * @param {any} document newly create document reponse from api post
+ * @returns {any} action and action types
+ */
+export function createDocumentSuccess(document) {
+  return {
+    type: types.CREATE_DOCUMENT_SUCCESS,
+    document
+  };
+}
+
+/**
+ *
+ * dispatch to reducer the selected document
+ * @export
+ * @param {any} id
+ * @returns {any} document id
+ */
+export function setCurrentDocument(id) {
+  return {
+    type: types.SET_CURRENT_DOCUMENT,
+    id
+  };
+}
+
+/**
+ *
+ * disptch to reducer the currently displayed document details
+ * @export
+ * @param {any} id
+ * @returns {any} document id
+ */
+export function displayCurrentDocument(id) {
+  return {
+    type: types.DISPLAY_SELECTED_DOCUMENT,
+    id
+  };
+}
+
+/**
+ * delete from state the currently selected document
+ * @return {[type]} [description]
```

Comment:
I would like to know the source of this @return type comment syntax. Kindly point me to it.